rollup.config.addon.js

```diff
@@ -15,11 +15,17 @@ export default (cliArgs) => {
     },
     plugins: [
       replace({
-        __ION_STUDIES_LIST__: cliArgs['config-study-list-url'] ?
-          `'${cliArgs['config-study-list-url']}'` :
-          "'https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/pioneer-study-addons-v1/records'",
+        // NOTE: if this URL ever changes, you will have to update the domain in
+        // the permissions in manifest.json.
+        __STUDIES_LIST__: cliArgs['config-studies-list-url'] ?
+          `'${cliArgs['config-studies-list-url']}'` :
+          "'https://firefox.settings.services.mozilla.com/v1/buckets/main/collections/rally-studies-v1/records'",
```

Comment:
Ah thanks for catching that, unfortunate that it needs to be duplicated... we don't *need* this right now since we support remote-settings but I think keeping it working is good since it's what we use for `build-local-addon` and eventually down the road we'll probably want to go unprivileged so this is good.